_posts/2023-05-06-reverse-engineering-gba.markdown

```diff
@@ -0,0 +1,1062 @@
+---
+layout: post
+title: Reverse Engineering a GBA Game
+date: 2023-05-06 05:00:00
+img: reverse-engineering/debug-env.png
+description: "How to reverse engineer a GBA game, using Pokémon Mystery Dungeon: Red Rescue Team as an example."
+author: Some Body
+---
+
+## Intro
+
+This tutorial will introduce the process of reverse engineering a game. We'll use the Game Boy Advance game _Pokémon Mystery Dungeon: Red Rescue Team_ for demonstration purposes, although most concepts discussed here can be applied to other games.
+
+### What is this tutorial about?
+This tutorial will walk you through the following:
+* Setting up a reverse engineering environment.
+* A basic introduction to assembly code.
+* Using the reverse engineering tool Ghidra to analyze assembly code.
```

Comment:
```suggestion
* Using the reverse engineering tool \"Ghidra\" to analyze assembly code.
```

Reply:
I feel quotes aren't needed here. It's common to use a description followed by a name.